src/Objs/editingConfigs.js

```diff
@@ -0,0 +1,6 @@
+function importAll(r) {
+  r.keys().forEach(r);
+}
+
+// import all *EditingConfig.js files under src/Objs/
+importAll(require.context("./", true, /EditingConfig\.js$/));
```

Comment:
* [ ] FYI this includes the `_*EditingConfig.js` shared helpers. An alternative could be sth like `/\w+\/\w+EditingConfig\.js$/`.

Reply:
Good point. But I think I would stick to the simpler approach here. The `_metadataEditingConfig.js` file for example would also be included in the bundle, if it would be moved in a subfolder of `src/Objs`. And both `_xEditingConfig.js` files are currently used indirectly in the editing config anyway.